internal/cmd/egctl/testdata/translate/out/from-gateway-api-to-xds.all.json

```diff
@@ -1,3 +1,4 @@
+[
```

Comment:
its an array since we have multiple `configKey`s ?

Reply:
Excuse me, I could not quite understand the words. It is already an array.

Reply:
yah so this was a bug, you fixed by adding the `[` and `]` ?

Reply:
Yes. The previous output is:
```
{}
{}
```
Now it become:
```
[
{},
{}
]
```